### ISSUE TITLE
Scheduler job refers to time in past in column "Scheduled for"

### ISSUE THREAD

**Issue by ralf401** [zammad/zammad#4927]
### Used Zammad Version

6.2

### Environment

- Installation method: source
- Operating system (if you're unsure: `cat /etc/os-release` ): fedora 39
- Browser + version: firefox 119.0

### Actual behaviour

The column "Scheduled for" in the scheduler management sometimes refers to a time in the past (e.g. 7 minutes ago), see screenshots of different examples:
![Screenshot from 2023-11-08 10-08-00](https://github.com/zammad/zammad/assets/60067527/96b04274-255d-40c6-b0f1-58bb89d9d0f3)
![Screenshot from 2023-11-08 10-03-40](https://github.com/zammad/zammad/assets/60067527/720f41f8-5b81-454a-985d-484b0bc65beb)
![Screenshot from 2023-11-08 10-21-59](https://github.com/zammad/zammad/assets/60067527/9d12922e-1e84-4f34-9719-4e3cb6495013)

### Expected behaviour

I would expect that the column "Scheduled for" only refers to a future execution of the job (e.g. in 5 minutes))

### Steps to reproduce the behaviour

Create a scheduler job :-) 

Selecting the execution time every 10 minutes might be helpful.

Unfortunately, I am not able to provide exact steps to reproduce this behavior, but it occurs quite often, so it should be possible to reproduce it.

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by MrGeneration** [comment]
Past dates can technically happen if the scheduler job didn't run to the moment you're checking.

**Comment by mantas** [comment]
Would it be better to show e.g. `Waiting for execution` instead of time in the past?

**Comment by ralf401** [comment]
Yes I think anything would be better than `X minutes ago` in the "Scheduled for" column. Fur sure, the best would be to only display time in the future. But if it's not possible, then `Waiting for execution` would be fine.

**Comment by cautopartes** [comment]
Just came across this behavior and searched and found this bug report. So here is the "bump".

I'd also go with  `Waiting for execution ` instead of insinuating a time travel to the user.

**Comment by dvuckovic** [comment]
Normally, when a scheduled appointment is imminent, the word `Due` may be shown. This puts it back to the proper context of the timestamp.

**Comment by vBenTec** [comment]
We've decided to show ➡️`due` if the scheduler's next run at couldn't be fulfilled at the moment of time. ℹ️ Additionally, we show the absolute timestamp on hover.
